test/scripts/e2e_subs/asset-misc.sh

```diff
@@ -18,7 +18,10 @@ ACCOUNTD=$(${gcmd} account new|awk '{ print $6 }')
 
 ASSET_NAME='Birlot : décollage vs. ࠶🦪'
 
-${gcmd} asset create --creator ${ACCOUNT} --name "${ASSET_NAME}" --unitname amisc --total 1000000000000
+# to ensure IPFS URLs longer than 32 characters are supported
+ASSET_URL="/ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/wiki/"
```

Comment:
Can you pad this out to 96 bytes?  We'd like it to fail if anyone ever gets the bad idea to shrink the limit, as that would break existing code (once this is out).

I realize that such an idea is so bad that it is unlikely to occur, but I still think that's what we should be testing.